test/dx_mgn_pool.js

```diff
@@ -66,6 +66,27 @@ contract("DxMgnPool", (accounts) => {
   })
 
   describe("participateInAuction()", () => {
+
+    it("non-owner can't ping it", async() => {
+      const dx = await DutchExchange.new()
+
+      const depositTokenMock = await MockContract.new()
+      const secondaryTokenMock = await MockContract.new()
+      const mgnTokenMock = await MockContract.new()
+      const dxMock = await MockContract.new()
+      const poolingEndBlock = (await web3.eth.getBlockNumber()) + 100
+      const instance = await DxMgnPool.new(depositTokenMock.address, secondaryTokenMock.address, mgnTokenMock.address, dxMock.address, poolingEndBlock)
+      
+      await depositTokenMock.givenAnyReturnBool(true)
+      
+      await dxMock.givenAnyReturnUint(42)
+      const postSellOrder = dx.contract.methods.postSellOrder(accounts[0], accounts[0], 0, 0).encodeABI()
+      const postSellOrderResponse = (abi.rawEncode(["uint", "uint"], [2, 0]))
+      await dxMock.givenMethodReturn(postSellOrder, postSellOrderResponse)
+
+      await instance.deposit(10)
+      await truffleAssert.reverts(instance.participateInAuction({from: accounts[2]})) //ownable has no designated error messages
```

Comment:
That's a shame. They should really add revert strings to all their requirements. Is it worth adding a ticket in their repo?

Reply:
Yeah, I will do it.